### PR TITLE
Add behat test.

### PR DESCRIPTION
**Pull request by mldk-tech**
#4

### CODE REVIEW

File: behat/features/login.feature [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Feature: Login
+  In order to be able to login to the student personal area
+  As an anonymous user
+  We need to be able to have access to the student personal area
+
+  @javascript
+  Scenario: Visit the login area page
+    Given I am on login page
+    When  I fill in "EmailOrUsername" with "boaz@eteachergroup.com"
+    And   I fill in "Password" with "Zaq12wsx"
```
Comment (mldk-tech) [inline behat/features/login.feature]: @amitaibu - It's that ok to insert the user name and the password in the scenario? or you prefer that the user name and password will be in a custom function ?

Comment (amitaibu) [inline behat/features/login.feature]:
``` gherkin
Given I login with with "boaz"`
When I visit "..."
I should see ""
```

**Comment by mldk-tech** [comment]
@amitaibu - Can you please confirm before I prepare a video?

File: behat/features/bootstrap/FeatureContext.php [hunk 2]
```diff
@@ -35,4 +35,54 @@ public function iShouldHaveAccessToThePage() {
...
+  /**
+   * @When I visit the student personal area page
+   */
+  public function iVisitTheStudentPersonalAreaPage2()
+  {
+    $this->getSession()->visit('https://student.eteachergroup.com/Login?ReturnUrl=%2f');
+
+    $element = $this->getSession()->getPage();
+    $element->fillField('EmailOrUsername', $this->user);
+    sleep(3);
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: why sleep?

File: behat/features/bootstrap/FeatureContext.php [hunk 3]
```diff
@@ -35,4 +35,54 @@ public function iShouldHaveAccessToThePage() {
...
+  {
+    $this->_login($name);
+  }
+
+
+  /**
+   * @When I visit the student personal area page
+   */
+  public function iVisitTheStudentPersonalAreaPage2()
+  {
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: move up

File: behat/features/bootstrap/FeatureContext.php [hunk 4]
```diff
@@ -35,4 +35,54 @@ public function iShouldHaveAccessToThePage() {
...
+    if ($name == "boaz") {
+      $this->user = "boaz@eteachergroup.com";
+      $this->password = "Zaq12wsx";
+    }
+  }
+
+  /**
+   * @Given I login with user :arg1
+   */
+  public function iLoginWithUser2($name)
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: ?

File: behat/features/bootstrap/FeatureContext.php (hunk 2 above)
Comment (mldk-tech) [inline behat/features/bootstrap/FeatureContext.php]: For the movie

File: behat/features/bootstrap/FeatureContext.php (hunk 4 above)
Comment (mldk-tech) [inline behat/features/bootstrap/FeatureContext.php]: You can enter more details?

File: behat/features/bootstrap/FeatureContext.php [hunk 5]
```diff
@@ -35,4 +35,53 @@ public function iShouldHaveAccessToThePage() {
...
+    $this->getSession()->visit('https://student.eteachergroup.com/Login?ReturnUrl=%2f');
+
+    $element = $this->getSession()->getPage();
+    $element->fillField('EmailOrUsername', $this->user);
+    sleep(3);
+    $element->fillField('Password', $this->password);
+    sleep(3);
+    $submit = $element->find('css',"#goButton input");
+
+    if (empty($submit)) {
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: remove

File: behat/features/bootstrap/FeatureContext.php [hunk 6]
```diff
@@ -35,4 +35,53 @@ public function iShouldHaveAccessToThePage() {
...
+      throw new \Exception(sprintf("No submit button at %s", $this->getSession()->getCurrentUrl()));
+    }
+
+    $this->clear();
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: remove

File: behat/features/bootstrap/FeatureContext.php [hunk 7]
```diff
@@ -35,4 +35,53 @@ public function iShouldHaveAccessToThePage() {
...
+    // Log in.
+    $submit->click();
+  }
+
+  public function clear() {
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: remove

**Comment by mldk-tech** [comment]
Hi @YuriGizra -

Can you please check if the test pass in the Docker?

**Comment by YuriGizra** [comment]
@amitaibu, @DavidKohav,
[link to video](https://drive.google.com/file/d/0ByKZo8kOe8BgSDRLQlhGTzhuZ0E/view?usp=sharing)

**Comment by amitaibu** [comment]
@YuriGizra in Gif format please.

**Comment by YuriGizra** [comment]
@amitaibu, @DavidKohav 
Gif:
e-teacher-shoov

**Comment by amitaibu** [comment]
@YuriGizra no, please send a link to the gif in http://gfycat.com/

**Comment by mldk-tech** [comment]
@amitaibu - http://gfycat.com/OrangeFarawayCleanerwrasse

**Comment by YuriGizra** [comment]
@amitaibu, link to gfycat:
http://gfycat.com/KindheartedOrnateCock

**Pull request closed without merging**